docs/db/changelog/changesets/powerauth-java-server/1.9.x/20241010-rest-client-caching.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.9.x/20241010-rest-client-caching.xml" author="Jan Pesek">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="pa_application_callback" columnName="timestamp_last_updated" />
+            </not>
+        </preConditions>
+        <comment>Add timestamp_last_updated column to pa_application_callback table.</comment>
+        <addColumn tableName="pa_application_callback">
+            <column name="timestamp_last_updated" type="timestamp(6)" defaultValueComputed="${now}">
+                <constraints nullable="false" />
```

Comment:
In the _push-server_ I have made it nullable. It should be consistent, I guess. Which way is preferred? https://github.com/wultra/powerauth-push-server/pull/885/files#diff-8253dccde67669ec19e57a75654708e5790bb76af59b316198034a83ada57aefR18